config/api.yml

```diff
@@ -703,6 +703,20 @@
     - :subcollection
     :methods: *70174834086080
     :klass: Dialog
+  :service_orders:
+    :description: Service Orders
+    :options:
+    - :collection
+    :methods: *70174834084700
+    :klass: ServiceOrder
+    :collection_actions:
+      :post:
+      - :name: create
+      - :name: edit
+      - :name: delete
+    :resource_actions:
+      :post:
+      - :name: edit
```

Comment:
could use the delete action at the resource level.

Also, I don't see any identifiers here.  Not in db/fixtures/miq_product_features.yml yet for entitling these to roles ?